src/components/Sidebar.tsx

```
@@ -1,18 +1,21 @@
-  ChartBarIcon,
-  CpuChipIcon,
   HomeIcon,
   ChatBubbleLeftRightIcon,
   DocumentTextIcon,
   XMarkIcon,
   Bars3Icon,
   LockOpenIcon,
   LockClosedIcon,
-  BookOpenIcon,
 } from '@heroicons/react/24/outline';
+import { BioPromptIcon as BioPromptIcon } from './ui/icons/BioPromptIcon';
```

Biome

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)